tests/plugins/clipboard/manual/pasteimage.md

```diff
@@ -1,10 +1,16 @@
 @bender-ui: collapsed
-@bender-tags: 4.6.2, bug, clipboard
+@bender-tags: 4.6.2, 4.17.0, bug, clipboard, 3757
 @bender-ckeditor-plugins: wysiwygarea, toolbar, undo, basicstyles, image, clipboard, sourcearea
 
 ## Scenario:
 
- * Paste image from your clipboard into the editor.
- 
-**Expected**: Image is pasted as base64 encoded file.
+1. Paste image from your clipboard into the editor.
```

Comment:
Our code style guide defines Expected and Unexpected as:
```md
**Expected**
```
So, without result word and just as bolded text, not the header. I know we have tests where there are different styles, but we should stick to the rules if there is no important reason to do it differently.

Also, please notice that with your style, there is an incorrect indentation for `Unexpected result`:
![image](https://user-images.githubusercontent.com/6498567/117101733-28135080-ad77-11eb-8545-f886fef9fd11.png)

Please, correct it according to: [code style rules](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests).